ensime_shared/vim_commands.py

```diff
@@ -0,0 +1,44 @@
+commands = {
```

Comment:
Yeah, let's put it in the `config.py` file and that's it. I don't feel strongly about this.